js/components/Checkbox.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+
+const Checkbox = ({ type = 'checkbox', name, checked = false, onChange }) => (
+  <input type={type} name={name} checked={checked} onChange={onChange} />
+);
+
+// Checkbox.propTypes = {
```

Comment:
Why do we have commented out code here?

Reply:
Will add prop-type lib and bring these code back. I think it is better to do prop-type check

Reply:
+1 I definitely want us to do prop-type validations!